docs/src/gs/create-workload-cluster.md

```diff
@@ -38,9 +42,12 @@ Run the command below to create a Kubernetes cluster with 1 control plane node a
 ```bash
 OCI_COMPARTMENT_ID=<compartment-id> \
 OCI_IMAGE_ID=<ubuntu-custom-image-id> \
-OCI_SHAPE=VM.Standard.E4.Flex \
-OCI_SHAPE_OCPUS=1 \
-OCI_SHAPE_MEMORY_IN_GBS= \
+OCI_CONTROL_PLANE_SHAPE=VM.Standard.E4.Flex \
```

Comment:
- [ ] Remove all of this and update docs to call out how to change the shape/ocpu/memory